pkg/krew/repos.go

```diff
@@ -9,7 +9,7 @@ const (
 
 //GetKrewIndexRepoName returns the krew-index repo name
 func GetKrewIndexRepoName() string {
-	override := os.Getenv("UPSTREAM_KREW_INDEX_REPO_NAME")
+	override := os.Getenv("INPUT_UPSTREAM_KREW_INDEX_REPO_NAME")
```

Comment:
this code actually runs on server side (IIRC), and do not have access to the GitHub inputs. what we will have to do is to accept this input, and then pass along this info to the backend as part of request.

the backend then needs to consider 3 scenarios:

- The override configured on server side. this should take preference over GitHub action input and defaults, as this helps with testing the server.
- The override configured as part of GitHub action (if no override provided on server side)
- The default values. (if no override provided at all)